modules/apps/27-interchain-accounts/types/validate.go

```diff
@@ -0,0 +1,41 @@
+package types
+
+import (
+	"regexp"
+	"strings"
+
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+)
+
+// DefaultMaxAddrLength defines the default maximum character length used in validation of addresses
+var DefaultMaxAddrLength = 128
+
+// IsValidAddr defines a regular expression to check if the provided string consists of
+// strictly alphanumeric characters
+var IsValidAddr = regexp.MustCompile("^[a-zA-Z0-9]*$").MatchString
+
+// ValidateVersion performs basic validation of the provided ics27 version string
```

Comment:
Maybe put a TODO: for adding a link to the spec (once the spec id is updated?)

Reply:
nite: It might be worth adding a very brief comment explaining what we're validating here. I almost forgot for a second myself :laughing:

Reply:
Done!